pymeasure/instruments/etslindgren/m7006_001.py

```diff
@@ -0,0 +1,154 @@
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+import logging
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class M7006_001(Instrument):
+    """Position control card of EMCenter modular system.
+    Allows for controll and feedback of the positioner card.
+    Comunication happes transparently through the EMCenter mainframe.
+    devices share the resource name of the mainframe, and
+    are identified by the slot number and the device number
+    (if applicable to the modual).
+    """
+
+    def __init__(self, resource_name, slot=1, device="A", **kwargs):
+
+        kwargs.setdefault("write_termination", "\n")
+        super().__init__(resource_name, "test", **kwargs)
+        self._slot = slot
+        self._device = device
+
+    def write(self, command):
+        super().write(f"{self._slot}{self._device}{command}")
+
+    def values(self, command, **kwargs):
+        return super().values(f"{self._slot}{self._device}{command}", **kwargs)
+
+    acceleration = Instrument.control(
+        "ACC?",
+        "ACC %g",
+        """Acceleration of the device in seconds. Settable 0.1 and 30,
+        and gettable.
+        """,
+        values=(0.1, 30),
+        validator=strict_range,
+    )
+
+    speed = Instrument.control(
+        "SPEED?",
+        "SPEED %g",
+        """Speed of the device, as a percentage of the max speed.
+        Settable 0.0 to 100.0, and gettable.""",
+        values=(0.0, 100.0),
+        validator=strict_range,
+    )
+
+    aux1 = Instrument.control(
+        "AUX1?",
+        "AUX1 %g",
+        """Auxiliary input 1, Settable to ON or OFF, and gettable.""",
+        values=("ON", "OFF"),
+        validator=strict_discrete_set,
```

Comment:
A better pattern is to call this property `aux1_enabled` and use `map_values=True` and `values={True: "ON", False: "OFF"}` -- then you can use Python booleans to more naturally interact with this.

Same below.